.gitignore

```diff
@@ -7,6 +7,7 @@ scratch
 node_modules
 dist
 .aider*
+.local
```

Comment:
this way you can put your local yaml configs in here and not worry about committing them.